package.json

```diff
@@ -10,6 +10,11 @@
       "types": "./lib/index.d.ts",
       "require": "./lib/index.js",
       "default": "./lib/esm/index.js"
+    },
+    "./lib": {
```

Comment:
I don't think we need this. Our tooling import things from `headers-polyfill` directly, and so should any consumer of this library.